msteams-platform/bots/what-are-bots.md

```diff
@@ -20,17 +20,17 @@ A bot is also referred to as a chatbot or conversational bot. It is an app that
 
 Conversational bots allow users to interact with your web service using text, interactive cards, and task modules.
 
-:::image type="content" source="../assets/images/invokebotwithtext.png" alt-text="Web service using text"lightbox="../assets/images/invokebotwithtext.png":::
+:::image type="content" source="../assets/images/invokebotwithtext.png" alt-text="The screenshot is an example that shows the Web service using text."lightbox="../assets/images/invokebotwithtext.png":::
 
-:::image type="content" source="../assets/images/invokebotwithcard.png" alt-text="web service using interactive cards"lightbox="../assets/images/invokebotwithcard.png"border="true":::
+:::image type="content" source="../assets/images/invokebotwithcard.png" alt-text="The screenshot is an example that shows the Web service using interactive cards."lightbox="../assets/images/invokebotwithcard.png":::
 
-:::image type="content" source="../assets/images/task-module-example.png" alt-text="web service using task module"lightbox="../assets/images/task-module-example.png"border="true":::
+:::image type="content" source="../assets/images/task-module-example.png" alt-text="The screenshot is an example that shows the Web service using task module."lightbox="../assets/images/task-module-example.png":::
```

Comment:
Incorrect alt-text. 
Also, see the above comment. https://github.com/MicrosoftDocs/msteams-docs/pull/6377/files#r937538189

Reply:
resolved